client/src/reducers/centroidLabels.js

```diff
@@ -0,0 +1,77 @@
+import calcCentroid from "../util/centroid";
+
+const initialState = {
+  labels: [],
+  toggle: false
```

Comment:
`toggle` is not a very good name - what does the state actually mean?

Is it `showCentroidLabels`?  Or?

Reply:
Renamed to `showLabels`